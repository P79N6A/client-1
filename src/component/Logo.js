import React, { memo } from "react";
import { makeStyles } from "@material-ui/styles";
export default memo(() => {
    // style
    const useStyles = makeStyles((theme) => ({
        logo: {
            height: 32,
            background: "rgba(255,255,255,.2)",
            margin: 16
        }
    }));
    const classes = useStyles();
    return React.createElement("div", { className: classes.logo });
});
//# sourceMappingURL=Logo.js.map