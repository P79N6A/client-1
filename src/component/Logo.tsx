/** @jsx jsx
 *  @description 小程序制作页
 *  @author 陈迎
 *  功能及完成度
 * */
import React, { memo } from "react";
import { css, jsx } from "@emotion/core";
export default memo(() => {
  // 样式
  const styles = {
    logo: css`
      height: 32px;
      background: rgba(255, 255, 255, 0.2);
      margin: 16px;
    `
  };

  return <div css={styles.logo} />;
});
