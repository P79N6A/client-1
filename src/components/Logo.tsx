import React, { memo } from "react";
import { css } from "@emotion/core";
import IconFont from "./IconFont";

const Logo: React.FC = memo(() => {
  const styles = {
    logo: css`
      height: 32px;
      width: 80px;
      text-align: center;
      font-size: 40px;
      margin-bottom: 28px;
    `
  };

  return (
    <div css={styles.logo}>
      <IconFont type="icon-logo" style={{ fontSize: 45 }} />
    </div>
  );
});

export default Logo;
