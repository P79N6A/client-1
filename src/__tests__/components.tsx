import React from "react";
import renderer from "react-test-renderer";
import Logo from "../components/Logo";

test("渲染Logo组件快照", () => {
  const component = renderer.create(<Logo />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
