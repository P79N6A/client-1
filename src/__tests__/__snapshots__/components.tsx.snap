// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`渲染Logo组件快照 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "65rrbh",
      "next": undefined,
      "styles": "
      height: 32px;
      width: 80px;
      text-align: center;
      font-size: 40px;
      margin-bottom: 28px;
    ",
    }
  }
>
  <i
    className="anticon"
    style={
      Object {
        "fontSize": 45,
      }
    }
  >
    <svg
      aria-hidden={true}
      className=""
      fill="currentColor"
      focusable="false"
      height="1em"
      width="1em"
    >
      <use
        xlinkHref="#icon-logo"
      />
    </svg>
  </i>
</div>
`;
